<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_8_1</class>
 <widget class="QDialog" name="fct_comp_8_1">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modification&amp;Suppression des reservations</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choisir un sp√©ctacle </string>
   </property>
  </widget>
  <widget class="QComboBox" name="noSpec_fct_combo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dateRep_fct_combo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoSpec</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DateRep</string>
   </property>
  </widget>
  <widget class="QComboBox" name="noRang_fct_combo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoRang</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoPlace</string>
   </property>
  </widget>
  <widget class="QComboBox" name="noPlace_fct_combo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choisir une place</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choisir un dossier</string>
   </property>
  </widget>
  <widget class="QComboBox" name="noDos_fct_combo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoDos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_fct_comp">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>751</width>
     <height>461</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>NoSpec</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DateRep</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>noPlace</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>noRang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DateEmTick          </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>libelleCat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>noDos</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>1031</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>libelleCat</string>
   </property>
  </widget>
  <widget class="QComboBox" name="libelleCat_fct_combo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 0);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>noSpec_fct_combo</zorder>
  <zorder>dateRep_fct_combo</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>noRang_fct_combo</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>noPlace_fct_combo</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>noDos_fct_combo</zorder>
  <zorder>label_10</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_11</zorder>
  <zorder>table_fct_comp</zorder>
  <zorder>label_2</zorder>
  <zorder>libelleCat_fct_combo</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lineEdit_1</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>lineEdit_6</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>noSpec_fct_combo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8_1</receiver>
   <slot>refreshDateRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateRep_fct_combo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8_1</receiver>
   <slot>refreshNoRang()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noPlace_fct_combo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8_1</receiver>
   <slot>refreshLibelleCat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noRang_fct_combo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8_1</receiver>
   <slot>refreshNoPlace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>libelleCat_fct_combo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8_1</receiver>
   <slot>refreshNoDos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8_1</receiver>
   <slot>Supprimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8_1</receiver>
   <slot>Modifier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshDateRep()</slot>
  <slot>refreshNoPlace()</slot>
  <slot>refreshResult()</slot>
  <slot>Modifier()</slot>
  <slot>refreshNoSpec()</slot>
  <slot>refreshNoRang()</slot>
  <slot>refreshLibelleCat()</slot>
  <slot>refreshNoDos()</slot>
  <slot>Supprimer()</slot>
 </slots>
</ui>
